agent/proxycfg/snapshot.go

```diff
@@ -706,12 +708,11 @@ type configSnapshotAPIGateway struct {
 	BoundGatewayConfig       *structs.BoundAPIGatewayConfigEntry
 
 	// Hosts is the list of extra host entries to add to our leaf cert's DNS SANs
-	Hosts       []string
-	AreHostsSet bool
+	Hosts []string
```

Comment:
Given the above docstring and the fact that we don't actually add any SANs to the leaf cert, I think this is unused as well, no? Can you confirm and drop as well?

Reply:
can do